doc/manual/cookbook.rst

```diff
@@ -429,6 +429,9 @@ Group analyses
 Group analysis is facilitated by morphing source estimates, which can be
 done *e.g.*, to ``subject='fsaverage'`` as::
 
-    >>> stc_fsaverage = stc.morph('fsaverage')  # doctest: +SKIP
+    >>> morph = mne.compute_source_morph(subject_from='sample',
+                                         subject_to='fsaverage',
+                                         src=stc)  # doctest: +SKIP
+    >>> stc_fsaverage = morph.apply(stc)  # doctest: +SKIP
```

Comment:
@larsoner this illustrates a current concern with the new API. You **must** provide a src parameter. 2 issues:

  - It cannot in fact be None although it's None in function signature.

  - you need to pass stc twice...

cc @TommyClausner

Reply:
Having to pass STC twice is fine by me. But we should not allow it to be none. One option is to actually make it the first argument

Reply:
+1 to make it the first argument with no default value. Can you push this to this branch if you have time?

Reply:
I can do it early next week